cfme/control/explorer/conditions.py

```diff
@@ -245,11 +247,11 @@ def exists(self):
 
         Returns: :py:class:`bool` signalizing the presence of the Condition in the database.
         """
-        conditions = self.appliance.db.client["conditions"]
-        return self.appliance.db.client.session\
-            .query(conditions.description)\
-            .filter(conditions.description == self.description)\
-            .count() > 0
+        try:
+            self.appliance.rest_api.collections.conditions.get(description=self.description)
```

Comment:
/me whispers Sentaku

Reply:
I entirely agree but this will be incorporated in another PR :)
I just got rid of db call. we shouldn't have them if we want devs use our code.